apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -161,7 +173,8 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                         prefetch={false}
                         className={classNames(
                           " bg-default dark:bg-muted border-default hover:bg-subtle hover:border-brand-default text-emphasis mb-2 block rounded-md border py-2 text-sm font-medium",
-                          brand === "#fff" || brand === "#ffffff" ? "" : ""
+                          brand === "#fff" || brand === "#ffffff" ? "" : "",
+                          isSlotReservationPending ? "pointer-events-none opacity-50" : ""
```

Comment:
According to Cal's design system this should be the styles of the disable state of the buttons.

```suggestion
                          reserveSlotMutation.isLoading ? "cursor-not-allowed opacity-40" : ""
```

the disabled attribute must be added.

`disabled={reserveSlotMutation.isLoading}`

Reply:
Yes, I tried this but `next/link` doesn't take disabled prop that is why I had to use pointer events to prevent user from redirecting to a different link while reservation mutation is happening.